dspace/config/spring/api/external-services.xml

```diff
@@ -92,5 +94,38 @@
         </property>
     </bean>
 
+    <bean id="scopusLiveImportDataProvider" class="org.dspace.external.provider.impl.LiveImportDataProvider">
+        <property name="metadataSource" ref="ScopusImportService"/>
+        <property name="sourceIdentifier" value="scopus"/>
+        <property name="recordIdMetadata" value="dc.identifier.scopus"/>
+        <property name="supportedEntityTypes">
+            <list>
+                <value>Publication</value>
+            </list>
+        </property>
+    </bean>
+
+    <bean id="pubmedEULiveImportDataProvider" class="org.dspace.external.provider.impl.LiveImportDataProvider">
+        <property name="metadataSource" ref="PubmedEuropeImportService"/>
+        <property name="sourceIdentifier" value="pubmedeu"/>
+        <property name="recordIdMetadata" value="dc.identifier.pmid"/>
+        <property name="supportedEntityTypes">
+            <list>
+                <value>Publication</value>
+            </list>
+        </property>
+    </bean>
+
+    <bean id="ciniiLiveImportDataProvider" class="org.dspace.external.provider.impl.LiveImportDataProvider">
+        <property name="metadataSource" ref="CiniiImportService"/>
+        <property name="sourceIdentifier" value="cinii"/>
+        <property name="recordIdMetadata" value="dc.identifier.other"/>
+        <property name="supportedEntityTypes">
+            <list>
+                <value>Publication</value>
+            </list>
+        </property>
+    </bean>
```

Comment:
If I'm not mistaken, I think we are missing a Web of Science configuration here?

Reply:
I think we are missing the Web of Science setup in this file, which is probably why it doesn't work.